### PR TITLE
Fix confirm string when remove nodes from Batch pool.

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

File: lib/commands/batch/batch.pool._js [hunk 1]
```diff
@@ -1055,7 +1055,7 @@ exports.init = function(cli) {
     }
     nodeList = interaction.promptIfNotGiven($('Nodes list: '), nodeList, _);
 
-    var tips = util.format($('Removing nodes from pool %s'), poolId);
+    var tips = util.format($('Removing node(s) \'%s\' from pool %s'), nodeList, poolId);
```
Comment (annatisch) [inline lib/commands/batch/batch.pool._js]: Does the 'nodeList' here need the .join(', ') that it has in the formatting below?

Comment (xingwu1) [inline lib/commands/batch/batch.pool._js]: No, nodeList is string.

**Comment by balajikris** [comment]
hey @xingwu1 @annatisch 
quick question here. This PR is against `Azure:dev` and I wanted to check if it was intentional. I usually see you guys commit to `Azure:batch-beta` or `Azure:batch-beta-dev`. So, I'm curious if this was really meant to be against `Azure:dev`. ?

**Comment by xingwu1** [comment]
@balajikris Yes, this is intentional. The batch-beta is using for some experiential features. However, this fix is  applied to official shipped cli v1.

**Comment by xingwu1** [comment]
@balajikris or @amarzavery, could you merge this change?

**Pull request merged**